plugins/ti-dmc/meson.build

```diff
@@ -0,0 +1,16 @@
+if gusb.found()
+cargs = ['-DG_LOG_DOMAIN="FuPluginTiDmc"']
+
+plugin_quirks += files('ti-dmc.quirk')
+plugin_builtins += static_library('fu_plugin_ti_dmc',
+  sources: [
+    'fu-ti-dmc-device.c',
+    'fu-ti-dmc-firmware.c',
```

Comment:
don't you want this as part of `# fuzzing`?

Reply:
I don't _think_ so. It's really just three blobs cat'd together, and I'm not sure it adds anything more than what we unit test `fu_bytes_new_offset()` already.